### ISSUE TITLE
Looking forward to an update of /physionet_databases/cinc2018.py /  

### ISSUE THREAD

**Issue by dddeanwang** [wenh06/database_reader#1]
Recently I tried to read this dataset but failed, fortunately I found your project but losing the "load_data()" func, hoping your update, thanks!

**Comment by wenh06** [comment]
So you are doing research on sleep analysis using ECG signals? I spent quite a lot of time on this topic some time ago, but not using CinC2018 data. Instead I used the dataset [SHHS](https://github.com/wenh06/database_reader/blob/master/database_reader/nsrr_databases/shhs.py). Currently I'm working on CinC2021, the unofficial phase. Maybe some time later I would update the CinC2018 reader.

**Comment by dddeanwang** [comment]
Yes, I'm doing some research on sleep apnea analysis with ECG signals. At present, I have run my model with Apnea-ECG and UCDDB datasets(thks for your resampled data), but these two datasets seem to be collected from different leads of ECG. I want to confirm whether the signals of different leads have effect on the apnea detection results. I am looking for another dataset to verify this idea, but it seems that there are not many ECG signal datasets with apnea annotations. I just found cinc2018, so that is the issue, so do you know some more datasets with apnea annotations? Would be much appreciated to receive your reply!

**Comment by wenh06** [comment]
your research problem is quite interesting.  as far as I know, despite the physionet datasets you mentioned, the NSRR datasets might have the annotations you need.

**Comment by wenh06** [comment]
> Yes, I'm doing some research on sleep apnea analysis with ECG signals. At present, I have run my model with Apnea-ECG and UCDDB datasets(thks for your resampled data), but these two datasets seem to be collected from different leads of ECG. I want to confirm whether the signals of different leads have effect on the apnea detection results. I am looking for another dataset to verify this idea, but it seems that there are not many ECG signal datasets with apnea annotations. I just found cinc2018, so that is the issue, so do you know some more datasets with apnea annotations? Would be much appreciated to receive your reply!

I now remember that I did use CinC2018 data, along with SHHS. These data are stored in the institute I left a few month ago. I tried to download the dataset using the torrent file provided by [Physionet](https://physionet.org/content/challenge-2018/1.0.0/), but failed.... I'll try downloading it from [GCP](https://console.cloud.google.com/storage/browser/challenge-2018-1.0.0.physionet.org?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false) using `gsutil` later some time.

**Comment by dddeanwang** [comment]
I tried to download cinc2018, failed either, it's too big that I only got part of the dataset, so I gave up using it.  With your guidance, I found SHHS and MESA dataset from NSSR might help, I'm still having a hard time tring to understand and extract the apnea information and annotations from it. Anyway, thanks for your help!
